### PR TITLE
Trustpilot block

### PR DESCRIPTION
**Pull request by axelbacklund**
Block with easily changeable Trustpilot star ratings and reviews

### CODE REVIEW

**Comment by axelbacklund** [comment]
Woops, branched from the not yet merged press card branch so those changes are here as well. Might delete this :)

**Comment by palmenhq** [comment]
@axelbacklund if you want to you can, until we merged it, change the base branch (if you press the edit button to the right of the headline) to be press-card-block and you will (hopefully) see only this branch's changes :)

**Comment by axelbacklund** [comment]
Merged the press card block now!

File: src/blocks/TrustpilotBlock/TrustpilotRatingItem.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+> = ({ color, title, rating }) => (
+  <>
+    <h4>{title}</h4>
+    <StarRatings
+      rating={Number(rating)}
+      starRatedColor={getColorStyles(color && color.color).background}
+      starEmptyColor="rgba(255, 255, 255, 0.2)"
+      starDimension="25px"
+      starSpacing="4px"
+      numberOfStars={5}
```
Comment (hedvigoscar) [inline src/blocks/TrustpilotBlock/TrustpilotRatingItem.tsx]:
```suggestion
      numberOfStars={11}
```

Comment (hedvigoscar) [reply src/blocks/TrustpilotBlock/TrustpilotRatingItem.tsx]: (just kidding)

**Pull request merged**